db/schema.rb

```diff
@@ -27,14 +27,13 @@
     t.string "address"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.boolean "status", default: true
   end
 
   create_table "employees", force: :cascade do |t|
     t.string "name"
     t.string "email"
     t.string "position"
-    t.string "private_number"
+    t.integer "private_number"
```

Comment:
I think it's not a good idea to touch this file. If you need to change the type of this column you should do that through a migration, let me know your thoughts on this.

Reply:
Oh, I don't remember touching that file, I made a migration from cmd but I will try again to see what happens

Reply:
oh yes! I can see the migration It's just above xD, my bad